### PR TITLE
Validation for Repository name

### PR DESCRIPTION
**Pull request by Sunandadadi**
Screen Shot 2022-09-23 at 3 46 08 PM
Screen Shot 2022-09-23 at 2 45 33 PM

### CODE REVIEW

File: src/components/modals/CreateRepoModalTemplate.tsx [hunk 1]
```diff
@@ -176,7 +191,7 @@ export default function CreateRepositoryModalTemplate(
               label="Repository name"
               isRequired
               fieldId="modal-with-form-form-name"
-              helperTextInvalid="Enter a repository name"
+              helperTextInvalid="Name must be alphanumeric, all lowercase and at most 255 characters long'"
```
Comment (HammerMeetNail) [inline src/components/modals/CreateRepoModalTemplate.tsx:194]: Can this be updated to include `-` and `_` characters too? The regex supports it, so something like `a-ok` or `a_ok` are valid. Also there's a trailing `'` in this message.

Comment (Sunandadadi) [reply src/components/modals/CreateRepoModalTemplate.tsx:194]: Yes, will update it.

Comment (HammerMeetNail) [reply src/components/modals/CreateRepoModalTemplate.tsx:194]: Oh, and maybe something about starting with an alphanumeric character.

**Review by HammerMeetNail — APPROVED** [review]
LGTM

**Pull request merged**